### PR TITLE
feat: use ethers

### PR DESCRIPTION
**Pull request by liuzjalex**
## What does this implement/fix? Explain your changes.

…

## Does this close any currently open issues?

If it fixes a bug or resolves a feature request, be sure to link to that issue.
…

## Pull request type

_Put an `x` in the boxes that apply_

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Where has this been tested?

…

## Any other comments?

…

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,8 +47,7 @@
   ],
   "dependencies": {
     "@conflux-dev/conflux-address-js": "^1.3.11",
-    "@ethersproject/abi": "^5.4.1",
-    "@ethersproject/transactions": "^5.4.0",
+    "ethers": "^5.6.0",
```
Comment (loaddss) [inline package.json:50]: 需要在代码中暴露 ethers 这个库，最后的用法类似 `import {ethers} from '@onekeyfe/blockchain-libs'`

File: src/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export type { StorageLike, ExternalConfig } from './types/external-config';
 export type { Wallet } from './types/wallet';
 export { BlockchainEngine } from './engine';
+export { ethers } from 'ethers';
```
Comment (liuzjalex) [inline src/index.ts:4]: 这样吗 @loaddss

**Review by huazhouwang — APPROVED** [review]

File: src/index.ts (hunk 2 above)
Comment (loaddss) [reply src/index.ts:4]: 这样写是没问题的，意思也是这个意思，就是得验证下，确认下是不是能获取到 ethers 的整个对象，如果自测过没问题就可以。

Comment (liuzjalex) [reply src/index.ts:4]: 测试过了，可以获取
`import { ethers } from '../../src';

test('Check ethers import/export properly', () => {
  expect(ethers.constants.AddressZero).toStrictEqual(
    '0x0000000000000000000000000000000000000000',
  );
});`

**Review by loaddss — APPROVED** [review]

**Pull request merged**